hooks/useConnectWallet.tsx

```diff
@@ -1,23 +1,10 @@
-import trackLoginPoints from "@/lib/stack/trackLoginPoints";
-import { useEffect } from "react";
-import { useAccount, useConnect } from "wagmi";
+import { usePrivy } from '@privy-io/react-auth';
 
 const useConnectWallet = () => {
-  const { connectors, connectAsync } = useConnect();
-  const connector = connectors[0];
-  const { address } = useAccount();
-
-  useEffect(() => {
-    if (!address) return;
-    trackLoginPoints(address);
-  }, [address]);
-
-  const connectWallet = () => connectAsync({ connector });
-
+  const { login } = usePrivy();
   return {
-    connectWallet,
-    address,
+    connectWallet: login,
   };
 };
 
-export default useConnectWallet;
+export default useConnectWallet;
```

Comment:
Please delete this hook. Directly use the `usePrivy` hook anytime you need `login`